share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/memory.param

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2013-2021 Axel Huebl, Rene Widera, Benjamin Worpitz, Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Define low-level memory settings for compute devices.
+ *
+ * Settings for memory layout for supercells and particle frame-lists,
+ * data exchanges in multi-device domain-decomposition and reserved
+ * fields for temporarily derived quantities are defined here.
+ */
+
+#pragma once
+#include <pmacc/math/Vector.hpp>
+#include <pmacc/mappings/kernel/MappingDescription.hpp>
+
+
+namespace picongpu
+{
+    /* We have to hold back 350MiB for gpu-internal operations:
+     *   - random number generator
+     *   - reduces
+     *   - ...
+     */
+    constexpr size_t reservedGpuMemorySize = 350 * 1024 * 1024;
+
+    /* short namespace*/
+    namespace mCT = pmacc::math::CT;
+    /** size of a superCell
+     *
+     * volume of a superCell must be <= 1024.
+     * This setup may use grid size that is multiple of 4 but not 8.
+     */
+    using SuperCellSize = typename mCT::shrinkTo<mCT::Int<4, 4, 4>, simDim>::type;
```

Comment:
Question: Can this become a problem for the arbitrary-order FDTD solver using more than 4 neighbours? Should this become at least the number of neighbours the solver uses? (We do not use more than 4 in this particular test, I am just wondering)

Reply:
This should be fine if the guard size is set to 2 or more: https://github.com/ComputationalRadiationPhysics/picongpu/blob/b17941d2b90afb83b17284cae1b70e62d33e088e/include/picongpu/param/memory.param#L70

Reply:
@steindev it is only centered for the 1.cfg configuration, as there is no simple way to have it centered regardless of grid size. I wrote in the .cfg file comments

Reply:
But in principle the default `memory.param` should determine the `SuperCellSize` or `GuardSize` on the basis of the number of neighbours used by the solver, right?

Reply:
> @steindev it is only centered for the 1.cfg configuration, as there is no simple way to have it centered regardless of grid size. I wrote it in the .cfg file comments

So the intent of the 8 GPU case really is to test whether PML runs on multiple MPI ranks, not whether it produces unexpected results when running on multiple ranks? However, I don't see a way to have a physics test on multiple MPI ranks without a second setup, either. :thinking:

Ohh maybe I do, Set the SupercellSize to `Int<2,2,2>` (if possible) and GuardSize to `Int<2,2,2>` as well. Then you can simulate the 60x60x60 volume with one or eight GPUs and keep the Arbitrary-oder solver working as well, right?

Reply:
Good idea @steindev , fixed. Also changed 8.cfg to 4.cfg since the setup also allows 2D case and so to not touch z supercells or MPI ranks